virttest/qemu_vm.py

```diff
@@ -898,11 +898,17 @@ def add_memorys(devices, params):
             """
             options = []
             mem_params = params.object_params("mem")
-            if mem_params.get("maxmem"):
+            mem_params.setdefault("mem", "512")
+            automem = mem_params.get("automem", "yes") == "yes"
```

Comment:
the memory may over commit for migration with the method, and I rem migration will failed when mem over commit